cmd/cluster-agent/app/app.go

```diff
@@ -244,10 +245,19 @@ func start(cmd *cobra.Command, args []string) error {
 	sc := clusteragent.ServerContext{
 		ClusterCheckHandler: clusterCheckHandler,
 	}
+
 	if err = api.StartServer(sc); err != nil {
 		return log.Errorf("Error while starting agent API, exiting: %v", err)
 	}
 
+	if config.Datadog.GetBool("cluster_agent.admission_controller.enabled") {
+		log.Info("Starting admission controller server")
+
+		if err = admissioncontroller.StartServer(); err != nil {
+			return log.Errorf("Error while starting admission controller server, exiting: %v", err)
+		}
+	}
+
 	wg := sync.WaitGroup{}
```

Comment:
This isn't your code but it would be a nice improvement
```suggestion
	var wg sync.WaitGroup
```